src/styles/mobile.scss

```diff
@@ -45,7 +45,6 @@
   .side-nav.side-nav--open .side-nav__open {
     top: 34px;
     margin-top: 0;
-    display: none;
```

Comment:
This `display` was set to `none` because the close button was visible in between the breakpoints, as mentioned [here](https://github.com/nodejs/nodejs.dev/pull/955#pullrequestreview-501592829)

Kinda seems unnecessary to me